stan/math/rev/functor/ode_adams.hpp

```diff
@@ -45,20 +45,24 @@ namespace math {
  * @return Solution to ODE at times \p ts
  */
 template <typename F, typename T_y0, typename T_t0, typename T_ts,
-          typename... T_Args>
+          typename... T_Args, require_eigen_vector_t<T_y0>* = nullptr>
```

Comment:
I think these most of the solver templates in this PR still need `require_eigen_col_vector_t<>`